jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -272,6 +285,8 @@ YUI.add('store', function() {
               <h2>Kubernetes</h2>
               {kubernetesButton}
             </div>
+            <span onClick={this._handleSearchClick}
```

Comment:
That would be because `_handleSearchClick` uses `evt.target` instead of `evt.currentTarget`, it doesn't look like changing it will be break the other links, but might be worth a quick check.